ietf/meeting/models.py

```diff
@@ -375,9 +375,8 @@ def updated(self):
         if self.schedule:
             assignments_updated = SchedTimeSessAssignment.objects.filter(schedule__in=[self.schedule, self.schedule.base if self.schedule else None]).aggregate(Max('modified'))["modified__max"]
         dts = [timeslots_updated, sessions_updated, assignments_updated]
-        if valid_only := [dt for dt in dts if dt is not None]:
-            return max(valid_only)
-        return None
+        valid_only = [dt for dt in dts if dt is not None]
+        return max(valid_only) if valid_only else None
```

Comment:
Consider:
```
dts = set([timeslots_updated, sessions_updated, assignments_updated])
dts.discard(None)
return max(dts) if len(dts)>0 else None
```

Reply:
(this is a pretty small style/readability nit)

Reply:
I'll merge this without the above suggestion - if you agree with it, please make another PR.

Reply:
TIL `discard`. Thanks. I'll see if a PR is necessary.